modules/katello_content_view.py

```diff
@@ -164,7 +162,8 @@ def main():
     )
 
     if not HAS_NAILGUN_PACKAGE:
-        module.fail_json(msg="Missing required nailgun module (check docs or install with: pip install nailgun")
+        module.fail_json(msg="Missing required nailgun module (check docs or install with: "
+                         "pip install nailgun")
```

Comment:
@akofink want to add a .travis.yml setup for that? I can turn it on. Either in this PR or separate.